mackerel-plugin-postgres/lib/postgres.go

```diff
@@ -20,7 +23,12 @@ var graphdef = map[string]mp.Graphs{
 		Unit:  "integer",
 		Metrics: []mp.Metrics{
 			{Name: "active", Label: "Active", Diff: false, Stacked: true},
-			{Name: "waiting", Label: "Waiting", Diff: false, Stacked: true},
+			{Name: "active_waiting", Label: "Active waiting", Diff: false, Stacked: true},
+			{Name: "idle", Label: "Idle", Diff: false, Stacked: true},
+			{Name: "idle_in_transaction", Label: "Idle in transaction", Diff: false, Stacked: true},
+			{Name: "idle_in_transaction_aborted_", Label: "Idle in transaction (aborted)", Diff: false, Stacked: true},
```

Comment:
Okay, I understand that it uses a simple normalization from postgresql state `idle in transaction (aborted)`. But I think we may drop the trailing underscore. (I would use simple switch case statements to reject unexpected states since nobody knows what happens in the future version)